### PR TITLE
Create a more efficient Semmle build.

### PR DESCRIPTION
**Pull request by michaelcfanning**

### CODE REVIEW

**Comment by michaelcfanning** [comment]
1. Continue to tinker with Semmle build (still not working, will solicit offline help)
2. Update all SLN dependencies to current versions, except for the explicit JSON.NET reference to 9.0
3. Update .NET core to 2.1

@chinarosesz @nickmarston @harleenkohli @lgolding

File: BuildPackagesFromLayoutDirectory.cmd [hunk 1]
```diff
@@ -41,8 +41,8 @@ goto :Exit
 xcopy /Y %LayoutForSigningDirectory%\net452\%1 %BinaryOutputDirectory%\net452\
 
 :: For .NET core, .exes are renamed to .dlls due to packaging conventions
-xcopy /Y %LayoutForSigningDirectory%\netcoreapp2.0\%~n1.dll %BinaryOutputDirectory%\netcoreapp2.0\
-xcopy /Y %LayoutForSigningDirectory%\netstandard2.0\%~n1.dll %BinaryOutputDirectory%\netstandard2.0\
+xcopy /Y %LayoutForSigningDirectory%\netcoreapp2.1\%~n1.dll %BinaryOutputDirectory%\netcoreapp2.1\
```
Comment (chinarosesz) [inline BuildPackagesFromLayoutDirectory.cmd]: >2.1 [](start = 47, length = 3)

No urgency, but we could use this opportunity to upgrade to 3.1. I remembered it wrong in the past, 2.2 is out of support and 2.1 is supported until January, 2021 [https://dotnet.microsoft.com/platform/support/policy/dotnet-core](https://dotnet.microsoft.com/platform/support/policy/dotnet-core)

File: src/ReleaseHistory.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## **v2.1.26** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.25) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.26) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.26) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.26)
+* PACKAGE BREAKING: Update tool directory to netstandard2.1, to reflect use of that version of .NET core.
```
Comment (chinarosesz) [inline src/ReleaseHistory.md]: >c [](start = 100, length = 1)

Nit: Core

File: BuildPackagesFromLayoutDirectory.cmd (hunk 1 above)
Comment (michaelcfanning) [reply BuildPackagesFromLayoutDirectory.cmd]: Got it, yes.

---
In reply to: [377832840](https://github.com/microsoft/sarif-sdk/pull/1750#discussion_r377832840) [](ancestors = 377832840)

**Pull request merged**